superset/reports/commands/execute.py

```diff
@@ -698,6 +711,6 @@ def validate(  # pylint: disable=arguments-differ
             self._model_id,
             self._execution_id,
         )
-        self._model = ReportScheduleDAO.find_by_id(self._model_id, session=session)
+        self._model = session.query(ReportSchedule).filter_by(id=self._model_id).first()
```

Comment:
When executing these reports async, we can't use the DAO here, as that uses the base filter that checks `g.user` which isn't set yet (later when we want to execute the report we need to know who the creator/modifier/owner was if we're not executing as the thumbs selenium user).

Reply:
nit: I tend to use `one_or_none` avoids sending unnecessary `LIMIT 1`

Reply:
Got it - will update accordingly 👍